Monika After Story/game/script-topics.rpy

```diff
@@ -3641,13 +3641,19 @@ label monika_trolley:
     m 1eua "Oh, cool. I love thinking about these sorts of thought experiments."
     m 1euc "I guess we're talking about real people, right? I wouldn't have a particular preference if I knew they didn't have free will."
     m 1dsc "Hmmm..."
-    m 1eua "If we were talking about the classic problem, of course I would switch."
-    m 1rkc "But if we're talking about real people..."
-    m 3esc "I wouldn't switch."
-    m 4esc "That's the only logical choice, if you think about it."
-    m 1euc "It feels nice to say that you'd save the most people, doesn't it? But people only say that because nobody in the example matters to them."
-    m "If they actually cared about the person, they'd kill the other five in a heartbeat."
-    m 2tsb "Do you know anyone who would want a self-driving car to save other people by killing the driver? Didn't think so, [player]."
+    m 3esa "The classic trolley problem has us choose between letting the trolley run over five people, or switching to the track where only one person will be killed."
+    m 1esa "The problem is mostly famous due to how divisive it is."
+    m "Regardless of whether they'd pull the switch or not, most people believe that their choice simply must be the correct one."
+    m 1esc "There are even some who'd say the right thing to do is not take part in this scenario at all."
```

Comment:
Again, I feel like you're singling out the people who have this stance, and dismissing it. 

Instead of criticizing other views, maybe you could have Monika mention the other views, with some of the reasoning behind them. 

For example, Two trains of thought for not pulling the lever are of course, not wanting to take direct responsibility for one person's death, or not wanting to pull because it isn't your place, and who are you to interfere with whatever is going on?

Reply:
I'm not trying to attack anyone here.
My intention here was to list the three stances one might take and point out that two of them are, for all intents and purposes, identical. Whether you choose to walk away from the problem, or to actively not pull the lever, the outcome is the same.
Looking at it from a lower level, I can see how the phrase `There are even some` might convey a bit of an accusatory tone.

Reply:
Yes, there are ultimately only 2 choices, and whatever you choose has to be one of the two. I agree that phrase makes it sound a bit more accusatory. 

Also, when Monika says "I don't think that really works", it almost sounds like she's saying it's the wrong course of action. There are 2 courses of action, but multiple different reasons behind each one, which is why I suggested you list a couple of the possible reasonings.

Reply:
The 'I don't think that really works' refers to the fact that you're the actor in this scenario and no matter what you do you can't reduce yourself to the role of a bystander, since even refusing to carry out the role of the actor ultimately fulfills it.
Okay, that's a bit convoluted, but I hope I'm making sense here.

Reply:
I would say something like "but even then, that's the same as not pulling the lever". 

My issue, is the way that it's worded, it specifically calls out the party that doesn't act, and makes it sound like she's calling their logic flawed, despite still being a viable option.

Reply:
Alright, noted. Before I make the changes, is there anything else that feels off to you? Like I said, I have trouble understanding the other side of the argument, so having someone actually point these thing out to me is probably the only way I'll notice.

Reply:
The only other thing that bugs me right now comes from how confusing the original dialogue is, so I think it's a good idea that the topic is being clarified and explained better. 

`m 1eua "If we were talking about the classic problem, of course I would switch."`
`m 1rkc "But if we're talking about real people..."`
`m 3esc "I wouldn't switch."`

The reason I asked the difference between the classic problem and what we already have comes from this bit here. I always thought the classic problem did assume real people, and nowhere in the topic does it actually specify that the one person on the other track was someone important at all, so I was confused what she meant by "real people", especially considering she assumes real people at the beginning, chooses to switch in the classic problem, but not to in the case of real people. Does the classic problem not assume real people?

Apart from that, I might also change that one line at the beginning about "free will" to "real people". I think that sounds a bit more polite/nicer in a general sense, and is easier to understand.

Reply:
Those three lines are actually the main reason I decided to rewrite the topic. The classic problem assumes real people afaik. Generic, and for the purpose of the problem undistinguishable, but real. I'd say the free will bit is her referencing the other girls being controlled by the script.

Reply:
If the bit about free will is specifically referencing the other girls, and only them, I think it's fine, but in a more general sense, such as if anyone else were to say it, I think it's easier to understand if she said "real people" instead. I think it serves the same purpose, and is easier to understand.

Reply:
I suppose it does feel more natural like that.